src/Maestro/DependencyUpdater/Program.cs

```diff
@@ -4,6 +4,9 @@
 
 using Maestro.Contracts;
 using Maestro.Data;
+using Maestro.RemoteFactory;
+using Microsoft.DotNet.DarcLib;
+using Microsoft.DotNet.Kusto;
```

Comment:
Sort these.  They are close, but not quite. :-)

Reply:
Looks right to me? 1 Maestro, then 2 Microsoft.DotNet, DarcLib, Kusto.

Reply:
```
using Microsoft.DotNet.Kusto;
using Microsoft.AspNetCore.Hosting;
```

It's not the ones you added, it's where you added them. :-)

Reply:
You're right. I'm wrong.